bentoml/_internal/cli/__init__.py

```diff
@@ -1,18 +1,32 @@
 import click
 
-from bentoml import __version__
+from bentoml import __version__ as BENTOML_VERSION
 
 from .yatai import add_login_command
 from .click_utils import BentoMLCommandGroup
 from .bento_server import add_serve_command
 from .containerize import add_containerize_command
+from ..configuration import is_pypi_installed_bentoml
 from .bento_management import add_bento_management_commands
 from .model_management import add_model_management_commands
 
 
+def _rich_callback():
+    import sys
+
+    if sys.stdout.isatty() and not is_pypi_installed_bentoml():
```

Comment:
does `sys.stdout.isatty` return True or False in a Jupyter notebook environment?

And why does this has to do with `is_pypi_installed_bentoml`?  I think regardless we should show a nice stacktrace when user is in an interactive mode, either a jupyter notebook or python session.

Reply:
I don't think we want to show traceback in non-development mode. This is more for us when developing CLI, Users can still see all the traceback from their end of the code.

`sys.stdout.isatty` return False in a jupyter notebook environment.